lib/tokenizers.ex

```diff
@@ -10,127 +10,4 @@ defmodule Tokenizers do
 
   This library has bindings to use pretrained tokenizers. Support for building and training a tokenizer from scratch is forthcoming.
```

Comment:
Is this line still up to date?

We should also add a note to consult the underlying modules. :)

Reply:
It is still up to date. Still working with pretrained only, but we'll be able to add training soon! Added the overview, lmk what you think.